src/redis-benchmark.c

```diff
@@ -1650,13 +1655,10 @@ int showThroughput(struct aeEventLoop *eventLoop, long long id, void *clientData
     config.previous_tick = current_tick;
     atomicSet(config.previous_requests_finished,requests_finished);
     printf("%*s\r", config.last_printed_bytes, " "); /* ensure there is a clean line */
-    int printed_bytes = printf("%s: rps=%.1f (overall: %.1f) avg_msec=%.3f (overall: %.3f)\r", config.title, instantaneous_rps, rps, hdr_mean(config.current_sec_latency_histogram)/1000.0f, hdr_mean(config.latency_histogram)/1000.0f);
-    if (printed_bytes > config.last_printed_bytes){
-       config.last_printed_bytes = printed_bytes;
-    }
+    config.last_printed_bytes = printf("%s: rps=%.1f (overall: %.1f) avg_msec=%.3f (overall: %.3f)\r", config.title, instantaneous_rps, rps, hdr_mean(config.current_sec_latency_histogram)/1000.0f, hdr_mean(config.latency_histogram)/1000.0f);
```

Comment:
maybe instead of changing that long line, it would be cleaner to keep the previous approach, store it into a var, and then just remove the `if` that we used to have and do the assignment unconditionally?
it'll certainly make the diff nicer.

alternatively, if we're already modifying this line in this PR anyway, we should probably break it into several lines.